.github/main.yml

```diff
@@ -0,0 +1,31 @@
+name: Continues integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: "npm"
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Install
+        run: npm ci
+      - name: Code style
+        run: npm run lint
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v4
+        with:
+          browser: chrome
+          headed: true
+      - name: Cypress run
+        uses: cypress-io/github-action@v4
+        with:
+          build: npm run build
+          start: npm start
```

Comment:
This section is not needed, you are already running cypress tests on the previous step :)